src/test/rpc/JSONRPC_test.cpp

```diff
@@ -2160,6 +2160,73 @@ class JSONRPC_test : public beast::unit_test::suite
             BEAST_EXPECT(RPC::contains_error(result));
         }
 
+        {
+            // 10: Call "sign" with nothing in the open ledger
+            Json::Value toSign;
+            toSign[jss::tx_json] = noop(env.master);
+            toSign[jss::secret] = "masterpassphrase";
+            auto rpcResult = env.rpc("json", "sign", to_string(toSign));
+            auto result = rpcResult[jss::result];
+
+            BEAST_EXPECT(!RPC::contains_error(result));
```

Comment:
That's not one of the relaxations :)

For example, tabs are not okay in test code...